src/System.Linq.Expressions/ref/System.Linq.Expressions.cs

```diff
@@ -1113,8 +1115,74 @@ public sealed partial class UnaryExpression : System.Linq.Expressions.Expression
         public System.Linq.Expressions.UnaryExpression Update(System.Linq.Expressions.Expression operand) { throw null; }
     }
 }
+namespace System.Linq.Expressions.Interpreter
```

Comment:
These are new APIs that weren't exposed before. What's the objective here? Should they be added?